spec/integration_spec.rb

```diff
@@ -80,6 +87,44 @@
     end
   end
 
+  context "using plain tcp connection" do
+    it "should successfully connect to the server if ssl explicitely disabled" do
+      expect {
+        Lumberjack::Client.new(:port => tcp_port, :host => host, :addresses => host, :ssl => false)
+      }.not_to raise_error
+    end
+
+    it "should fail to connect to the server if ssl not explicitely disabled" do
+      expect {
+        Lumberjack::Client.new(:port => tcp_port, :host => host, :addresses => host)
+      }.to raise_error(RuntimeError, /Must set a ssl certificate/)
+    end
```

Comment:
We should have a test when a **client** with SSL disabled connect to a SSL Server. 
I think the code will retry to connect indefinitely, see https://github.com/elastic/ruby-lumberjack/pull/14/files#diff-e02f5a53687afb84cbe78c42e99f189fL60 and I don't think this is the expected user error, this should be fatal.